src/components/MethodField.vue

```diff
@@ -24,18 +27,21 @@ export default {
         return {
             transferAmount: null,
             transferTo: null,
+            expand: false,
         }
     },
     mounted() {
         this.setValues();
     },
     methods: {
+        toggle(){
+            this.expand = !this.expand;
+        },
         async setValues() {
             if (!this.trx.parsedTransaction)
                 return;
 
-            if (this.trx.parsedTransaction.sighash === ERC20_SIGHASH && this.contract) {
-                console.log(this.trx.parsedTransaction);
+            if (TRANSFER_SIGNATURES.includes(this.trx.parsedTransaction.sighash) && this.contract && this.contract.token && this.contract.token.decimals) {
```

Comment:
`this.contract && this.contract.token && this.contract.token.decimals` -> `this?.contract?.token?.decimals`